src/css/Navbar.module.css

```diff
@@ -4,20 +4,42 @@
   justify-content: space-between;
   padding-right: 20px;
   padding-left: 20px;
+  height: 5.938rem;
+  background-color: #fff;
+  margin-bottom: 0.938rem;
 }
 
 .nav {
   display: flex;
-  gap: 10px;
   list-style: none;
+  align-items: center;
+  padding-left: unset;
 }
 
 .nav a {
-  color: #1295d6;
   border-right: 2px solid black;
   padding-right: 10px;
+  font-family: Montserrat, sans-serif;
+  font-size: 1rem;
+  color: #121212;
+  text-decoration: none;
+  opacity: 0.7;
 }
 
 .nav a:last-child {
   border: unset;
 }
+
+#title {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
```

Comment:
Great job @ibugithub 🤝

- [ ] As stated in the project requirements, kindly ensure there is no repeating of the same rule. Instead, create a single class to represent this rule and reuse this class where necessary. For example make sure the following classes are not repeated. `font-family: Montserrat, sans-serif;`, `display: flex`, etc. You may implement this across all your stylesheets.